README.md

```diff
@@ -1601,6 +1601,7 @@ _See also [Documentation Generators](#documentation-generators), [Wikimatrix](ht
 - [DietPi](http://dietpi.com/) - Minimal Debian OS optimized for single-board computers, which allows you to easily install and manage several services for selfhosting at home. ([Source Code](https://github.com/Fourdee/DietPi)) `GPL-2.0` `Shell`
 - [DockSTARTer](https://dockstarter.com/) - DockSTARTer helps you get started with home server apps running in Docker. ([Source Code](https://github.com/GhostWriters/DockSTARTer)) `MIT` `Shell`
 - [DPlatform](https://dfabric.github.io/DPlatform-Shell/) - Deploy self-hosted apps easily: simple, bloat-free, independent installation. ([Source Code](https://github.com/j8r/DPlatform)) `MIT` `Shell`
+- [EmbassyOS](https://github.com/Start9Labs/embassy-os) - One-click self-hosting over TOR. Graphical operating system designed to facilitate the discovery, installation, configuration, private self-hosting, and reliable operation of open-source software services and applications. It aims to eliminate trust and custodianship from personal computing. `Rust`
```

Comment:
Please move EmbassyOS to non-free.md (https://github.com/Start9Labs/embassy-os/blob/master/LICENSE.md)

Reply:
True. But the license limits redistribution and fields of endeavor:

>the License does NOT grant you the right to:
>    Sell the Software or any derivative works based thereon.
>    Distribute the Object Code.

- https://en.wikipedia.org/wiki/Free_software#Definition_and_the_Four_Essential_Freedoms_of_Free_Software
- https://en.wikipedia.org/wiki/Debian_Free_Software_Guidelines

Hence it's non-Free

Reply:
https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2461